src/atoms/buttons/button.test.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { render, wait } from "@testing-library/react";
+import { Button } from "./button";
+import faker from "faker";
+import { ButtonSizes } from "../constants/button-sizes";
+import { ButtonStyles } from "../constants/button-styles";
+
+describe("Button", () => {
+    it("when default props, renders button children", async () => {
+        // Arrange
+        const expected = faker.random.words();
+
+        // Act
+        const { getByText } = render(<Button>{expected}</Button>);
+
+        // Assert
+        await wait(() => {
+            expect(getByText(expected)).not.toBeNull();
```

Comment:
Question- will `.not.toBeNull()` perform a nullsy check? ie, if `children` is undefined, would it catch that?

Reply:
Good idea, i'll verify

Reply:
Yeah, double checked that this fails if children isn't passed.